sdk/src/main/java/io/dapr/actors/AbstractClientBuilder.java

```diff
@@ -2,48 +2,46 @@
  * Copyright (c) Microsoft Corporation.
  * Licensed under the MIT License.
  */
-
 package io.dapr.actors;
 
-import okhttp3.OkHttpClient;
-import io.dapr.actors.*;
-
 /**
  * Base class for client builders
  */
 public abstract class AbstractClientBuilder {
 
-    /**
-     * Default port for Dapr after checking environment variable.
-     */
-    private int port = AbstractClientBuilder.GetEnvPortOrDefault();
-
-    /**
-     * Overrides the port.
-     * @param port New port.
-     * @return This instance.
-     */
-    public AbstractClientBuilder withPort(int port) {
-        this.port = port;
-        return this;
+  /**
+   * Default port for Dapr after checking environment variable.
+   */
+  private int port = AbstractClientBuilder.GetEnvPortOrDefault();
```

Comment:
Any particular reason to move from 4 spaces to 2 spaces? I am good either way but sticking to 4 spaces would reduce the diff :)

Reply:
I'd like to preserve horizontal space. If you can agree, I'd prefer. Unfortunately, I had two commits one with small changes and one more with the formatting only. But I created a patch to follow the rules for pull requests and patched my forked copy of the repo. Therefore the commit is huge.